src/librustc_driver/pretty.rs

```diff
@@ -761,6 +793,7 @@ pub fn pretty_print_input(sess: Session,
             debug!("pretty printing flow graph for {:?}", opt_uii);
             let uii = opt_uii.unwrap_or_else(|| {
                 sess.fatal(&format!("`pretty flowgraph=..` needs NodeId (int) or
+                                     \
```

Comment:
This is weird - could you file a rustfmt issue and do a manual fixup here please?